git/git.go

```diff
@@ -28,56 +26,6 @@ func HeadCommit(path string) (string, error) {
 	return strings.Trim(string(output), "\n"), err
 }
 
-func formatGitURL(endpoint, branch string) string {
```

Comment:
I moved these two functions so that they could be attached to a struct so that I could reuse an http client, as opposed to creating a new client on each request, as it was previously doing.